client/landing/stepper/declarative-flow/site-migration-flow.ts

```diff
@@ -134,31 +134,46 @@ const siteMigration: Flow = {
 			const siteId = getSiteIdBySlug( siteSlug );
 
 			switch ( currentStep ) {
-				case 'import': {
-					return navigate( 'waitForAtomic', { siteId, siteSlug } );
+				case STEPS.SITE_MIGRATION_SOURCE.slug: {
+					return navigate( STEPS.SITE_MIGRATION_PLUGIN_INSTALL.slug, {
+						siteId,
+						siteSlug,
+					} );
+				}
+
+				case STEPS.SITE_MIGRATION_PLUGIN_INSTALL.slug: {
+					if ( providedDependencies?.error ) {
+						return navigate( STEPS.ERROR.slug );
+					}
+
+					return navigate( STEPS.SITE_MIGRATION_INSTRUCTIONS.slug );
```

Comment:
Can we update the site migration steps test suite with the flows, it will not break it on future changes and reducing the PR review time

Reply:
The test was already updated https://github.com/Automattic/wp-calypso/pull/87610/files#diff-45224004cbff6a36fca585198ec66d76c87a421023e5ec3eb1788605b24e2f66.